assets/src/scripts/components/graph/state/options.js

```diff
@@ -0,0 +1,88 @@
+import memoize from 'fast-memoize';
+import { createSelector } from 'reselect';
+
+const MOUNT_POINT = 'components/graph/options';
+const LITHOLOGY_VISIBILITY_SET = `${MOUNT_POINT}/LITHOLOGY_VISIBILITY_SET`;
+
+const DEFAULT_LITHOLOGY_VISIBILITY = true;
```

Comment:
Should we be defaulting this off? In my view it's better to have less on a graph for normal folk like me so that we are not overwhelmed by information.